### PR TITLE
Add Continuous Deployment code to this repo

### LINKED ISSUE #22: Continuous deployment
A goal of issue https://github.com/raiden-network/raiden-pathfinding-service/issues/82 was to create a continuous deployment pipeline for the PFS. This makes sense for the MS as well.

This is blocked by the deployment of the MS contract on the testnets.

The current setup is at https://github.com/palango/services-dev
### Tasks

- [ ] Setup deployment pipeline

### PR DESCRIPTION
**Pull request by karlb**
Merge https://github.com/palango/services-dev into this repo.

* add containers for MS and RC
* use single image for all containers
* provide `docker-compose.override.yml` for local testing
* reuse shared args via yaml merge command
* use keystore (which is now required for all services)

Closes #22

### CODE REVIEW

File: deployment/.gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+venv/
```
Comment (palango) [inline deployment/.gitignore]: should we have this file here?

File: deployment/builder/app/main.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        os.chdir(source)
+        _print('git fetch')
+        subprocess.check_output(["git", "fetch", "--all"])
+        _print('git reset')
+        subprocess.check_output(["git", "reset", "--hard", f"origin/{branch}"])
+
+        _print(f'Switching to {deployment}')
+        os.chdir(deployment)
+        _print('docker build')
+        subprocess.check_output(["docker-compose", "build"])
```
Comment (palango) [inline deployment/builder/app/main.py]: Do we need to update this?

Comment (palango) [reply deployment/builder/app/main.py]: nvm

File: deployment/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  ms-rinkeby:
+    << : *defaults
+    command: ["python3", "-m", "monitoring_service.cli"]
+    environment:
+      - MS_ETH_RPC=http://geth.rinkeby.ethnodes.brainbot.com:8545
+      - MS_REGISTRY_ADDRESS=0xE4769320245E4DeEAE5e816C9A827098B2988ef8
+      - MS_MONITOR_CONTRACT_ADDRESS=0xA0ED03b276cA581e8F120b5DcFe152DaAb0162d9
+      - MS_STATE_DB=/state/ms-rinkeby.db
+
+  # TODO: wrong addresses!
```
Comment (palango) [inline deployment/docker-compose.yml]: We should do this automatically, but will this even start?

Comment (karlb) [reply deployment/docker-compose.yml]: Yes, we will only get problems once we do transactions on those contracts.

File: deployment/.gitignore (hunk 1 above)
Comment (karlb) [reply deployment/.gitignore]: No, that only made sense in the separate repo.

**Pull request merged**